### PR TITLE
Multiple Vulnerabilities in RubyGems

### PR DESCRIPTION
**Pull request by unak**

### CODE REVIEW

File: en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+The following vulnerabilities have been reported.
+
+* Prevent path traversal when writing to a symlinked basedir outside of the root.
+* Fix possible Unsafe Object Deserialization Vulnerability in gem owner.
+* Strictly interpret octal fields in tar headers.
+* Raise a security error when there are duplicate files in a package.
+* Enforce URL validation on spec homepage attribute.
+* Mitigate XSS vulnerability in homepage attribute when displayed via gem server.
+* Prevent Path Traversal issue during gem installation.
```
Comment (JuanitoFatas) [inline en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md]: Do any of above has CVE number? Maybe we should reference their CVE number.

File: en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+## Affected Versions
+
+* Ruby 2.2 series: 2.2.9 and earlier
+* Ruby 2.3 series: 2.3.6 and earlier
+* Ruby 2.4 series: 2.4.3 and earlier
+* prior to trunk revision 62422
+
+## Workarounds
+
+If you can't upgrade Ruby itself, upgrade RubyGems to the latest version.
```
Comment (JuanitoFatas) [inline en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md]: > If you can't upgrade Ruby itself

Do we have newer Ruby version that bundles newer rubygems?

File: en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+About the trunk, update to the latest revision.
+
+## Credits
+
+This report is based on [the official blog of RubyGems](http://blog.rubygems.org/2018/02/15/2.7.6-released.html).
+
+## History
+
+* Originally published at 2017-02-17 03:00:00 UTC
```
Comment (JuanitoFatas) [inline en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md]: 2017-02-17 should be 2018.

File: en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md (hunk 1 above)
Comment (unak) [reply en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md]: They (RubyGems team) does not get CVE numbers now.

File: en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md (hunk 2 above)
Comment (unak) [reply en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md]: oops

File: en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md (hunk 3 above)
Comment (unak) [reply en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md]: oops

**Pull request merged**